### PR TITLE
The dev startup scripts now supports multiple containers in dev mode

### PR DESCRIPTION
**Pull request by danakim**

### CODE REVIEW

**Comment by quartje** [comment]
On the last line a closing " is missing which makes it does not work. If I add it, it works fine.

Two remarks:
Some apps are still missing:
- azuremfa
- tiqr

The rest I've put inline in the script

File: stepup/start-dev-env.sh [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
+    app=$(echo $arg | cut -d ':' -f 1)
+    path=$(echo $arg | cut -d ':' -f 2)
+    echo "export ${app^^}_CODE_PATH=${path}" >> .start-dev-env-vars
+    docker_compose_args+=("-f ./${app}/docker-compose.override.yml")
+done
 
-# Use docker compose to start the environment but with the modified override file
+# Read the generated env file with the apps and their code paths
+source .start-dev-env-vars ; rm .start-dev-env-vars
+env
```
[the remark this answers is not in the document]
Comment (danakim) [reply stepup/start-dev-env.sh:14]: Ah yes, this is a leftover :) Sorry, thanks for catching it.

File: stepup/start-dev-env.sh [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
+
+# Use docker compose to start the environment but with the modified override file(s)
 echo -e "Starting the dev environment with the following command:\n"
-echo -e "docker compose -f docker-compose.yml -f ./${SERVICE}/docker-compose.override.yml up "${@:3}"\n"
-docker compose -f docker-compose.yml -f ./${SERVICE}/docker-compose.override.yml up "${@:3}"
+echo -e "docker compose -f docker-compose.yml ${docker_compose_args[@]} up "${@:3}"\n"
+docker compose -f docker-compose.yml ${docker_compose_args[@]} up "${@:3}
```
[the remark this answers is not in the document]
Comment (danakim) [reply stepup/start-dev-env.sh:19]: Thanks, will close it.

File: stepup/start-dev-env.sh [hunk 3]
```diff
@@ -1,10 +1,19 @@
 #!/usr/bin/env bash
 
-# Read the command line parameters to configure the dev env
-export SERVICE=$1
-export CODE_PATH=$2
+# Read the apps and their code paths from the arguments passed to the script
+docker_compose_arg=()
```
[the remark this answers is not in the document]
Comment (danakim) [reply stepup/start-dev-env.sh:4]: @quartje - this is used 5 lines lower where I add to this array. Here I initialise it (empty) and add to it later in the for loop.

**Comment by danakim** [comment]
@quartje - I have fixed these in commit 8d85243aad368ea5b7c80c98e1be3c657a53aa62. Will resolve the issues.

**Comment by danakim** [comment]
@quartje - also have added the missing applications in commit: d15304761246cea33fb7fc8b6942ba67f2eaad18

**Comment by quartje** [comment]
This seems to work! Let's merge & roll!

**Comment by danakim** [comment]
@quartje - thanks a lot for testing! Merging now.

**Pull request merged**